timemachine/ff/handlers/nonbonded.py

```diff
@@ -391,6 +415,22 @@ def static_parameterize(mol):
 
 
 class AM1CCCHandler(SerializableMixIn):
+    """The AM1CCCHandler stands for AM1 Correctable Charge Correction (CCC) which uses OpenEye's AM1 charges[1]
+    along with corrections provided by the Forcefield definition in the form of smirks and charge deltas.
+
+    This handler supports back propagation with respect to ligand parameters, which is what the "Correctable" refers
```

Comment:
I'd simplify this to simply "This handler supports jax.grad with respect to the forcefield parameters".